index.js

```diff
@@ -4,13 +4,19 @@ if (!process.env.TOKEN) {
 
 const {
     TOKEN,
-    USERNAME = ``,
+    BOT_USERNAME = ``,
     LOG_CHANNEL,
+    API_ROOT,
 } = process.env;
 
-const Telegraf = require(`telegraf`);
+const Telegraf = require(`telegraf/telegraf`);
 const bot = new Telegraf(TOKEN, {
-    username: USERNAME,
+    username: BOT_USERNAME,
+    telegram: {
+        ...API_ROOT && {
+            apiRoot: `http://tgp.sh-sh.ir/`,
```

Comment:
I think you forgot to change this to `API_ROOT`.

Reply:
I didn't have enough focus on those days :joy: sorry for that

Anyway, I'm back at home and will fix these now.